downloads/swan-lake-release-notes/swan-lake-beta1/RELEASE_NOTE.md

```diff
@@ -38,5 +38,9 @@ If you have not installed jBallerina, then download the [installers](/downloads/
 - 
 
 #### Developer Tools
-- 
+
+##### Debugger
+
+- Added remote debug support with the executable JAR.
```

Comment:
```suggestion
- Added remote debugging support for Ballerina executable JAR run command.
```

Reply:
How about saying "Added remote debugging support for the creation/generation of the Ballerina executable JAR." since debugging support is not for the run command?

Reply:
@praneesha Remote debugging support is not related to the creation/generation of the Ballerina executable JAR like you have mentioned. It is bound with the ballerina command which runs the created JAR.

Reply:
Then, how about saying: "Added remote debugging support for the command, which runs the Ballerina executable JAR."?

Reply:
+1

Reply:
Fixed with https://github.com/ballerina-platform/ballerina-dev-website/pull/2197/commits/0b5acefc50bac6542f271cb41574a1402d95fd0d